djangocon_2020/content/conduct/code_of_conduct/code_of_conduct.md

```diff
@@ -73,41 +73,45 @@ In case of a Code of Conduct violation, some of the most common actions organise
+Please do not feel like you may be a burden to us by reporting incidents. Even if you happen to report multiple incidents during the conference. We rather consider reports as an opportunity for us to act: by knowing about an incident, we can act on it, and often prevent it from going on or occuring again. But if we don't know, we can't take action.
+
+If you're not sure if an incident was already reported by someone else, even if many people saw it, please report it. It is frequent that several people witness an incident and no one reports it, because everyone thought other people would.
 
 If you are not sure whether the situation was a Code of Conduct violation, or whether it applied to that particular space, we encourage you to still report it. We would much rather have a few extra reports where we decide to take no action, rather than miss a report of an actual violation. We do not look negatively on you if we find the incident is not a violation. And knowing about incidents that are not violations, or happen outside our spaces, can also help us to improve the Code of Conduct or the processes surrounding it.
 
+If you think you witness something, but you're not sure what you saw, or you don't have proof, it's still very useful to us if you can report on that. You don't have to build up a case before reporting, and it's ok to report things even if you have doubts.
+
 In your report please include, when possible:
 
 * Your contact info (so we can get in touch with you)
 * Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
 * When and where the incident occurred. Please be as specific as possible.
-* Your account of what occurred. If there is a written record (e.g. tweets or slack messages) please include screenshots, or otherwise a link.
+* Your account of what occurred. If there is a written record (e.g. tweets or slack messages), please include screenshots and, if possible, a link.
 * Any extra context relevant for the incident.
 * If this incident is ongoing.
 * Any other information you would like to give us.
 
-If you don't have some of this information, or not at this time, please still make the report anyways.
+If you don't have some of this information, or not at this time, please still make the report anyways. You can contact us at any time after your report if you want to add, edit or take back any information you shared.
 
-If you feel unsafe reporting in person, you may choose someone to represent you. In this case, we'd need their contact information, but we'd ask you to make clear that this person represents you.
+If you feel unsafe reporting in person, you may choose someone to represent you. In this case, we'd need their contact information, but we'd ask this person to make clear that they are not reporting in their own name.
```

Comment:
Not sure at all:
- Does the CoC team need to know the identity of the victim ?
- Or does it need to know that the reporter is not the victim ?

Reply:
I think we should not oblige anyone to be identified, in order not to discourage people from reporting any problem they might have. However, it is important to be 100% clear that the reporter is not the victim.